homeassistant/components/elgato/light.py

```diff
@@ -143,12 +143,15 @@ async def async_update(self) -> None:
             self._state = None
 
     @property
-    def device_info(self) -> dict[str, Any]:
+    def device_info(self) -> DeviceInfo | None:
```

Comment:
```suggestion
    def device_info(self) -> DeviceInfo:
```

This can't be `None`